example/basic_robot_control.py

```diff
@@ -0,0 +1,81 @@
+import time
+
+import numpy as np
+
+from home_robot.motion.stretch import STRETCH_HOME_Q, HelloStretch
+from home_robot_hw.stretch_client import StretchClient
+
+if __name__ == "__main__":
+    robot = StretchClient()
+    model = HelloStretch()
+
+    # Acquire camera observations
+    imgs = robot.camera.get_images()
+
+    # Get camera pose
+    camera_pose = robot.camera.get_pose()
+    print(f"camera_pose={camera_pose}")
+
+    # Move camera
+    robot.camera.set_pan_tilt(pan=np.pi / 4, tilt=-np.pi / 3)
+    time.sleep(1)
+    robot.camera.look_at_ee()
+    time.sleep(1)
```

Comment:
do we need these sleeps? should we fix "wait"

Reply:
We have wait implemented, wonder why I didn't think of using those.